### PR TITLE
Update LCDInfo

### PR DESCRIPTION
**Pull request by Godimas**
Adding in fusion reactors. I copied the hydrogen engine code (since the fusion reactors are based on those) and changed the names

### CODE REVIEW

**Review by Beersatron — CHANGES_REQUESTED** [review]
You probably need to init the fusionReactor object. I would probably also call it fusionReactors (plural) so it is in line with the others.

List<IMyFusionReactor> fusionReactors = new List<IMyFusionReactor>();

**Comment by Beersatron** [comment]
As for dumping the tank names, try this inside the DrawTankSprites function. You might need to tinker with the path for getting the names from the objects.

`foreach (var tank in tanks) {
WriteTextSprite(ref frame, tank.BlockDefinition.SubtypeName, position, TextAlignment.LEFT);
position += newLine;
}`

File: Other Goodies/Mods/LCDInfo [hunk 1]
```diff
@@ -299,6 +306,26 @@ namespace EconomySurvival.LCDInfo
 
             position += newLine + newLine;
         }
+		
+        void DrawFusionReactorSprite(ref MySpriteDrawFrame frame, ref Vector2 position, IMyTextSurface surface)
+        {
```
Comment (Godimas) [inline Other Goodies/Mods/LCDInfo:311]: this is shit, make it less shit

Comment (Godimas) [reply Other Goodies/Mods/LCDInfo:311]: MAKE ME SKINI!

**Review by Godimas — COMMENTED** [review]
changes suggested

**Pull request merged**